core/jobs_registry.py

```diff
@@ -54,7 +54,33 @@
     exp_jobs_one_off.ImagesAuditJob,
     exp_jobs_one_off.VerifyAllUrlsMatchGcsIdRegexJob,
     exp_jobs_one_off.TranslatorToVoiceArtistOneOffJob,
-    prod_validation_jobs_one_off.ProdValidationAuditOneOffJob,
+    prod_validation_jobs_one_off.ActivityReferencesModelAuditOneOffJob,
```

Comment:
Btw, @ankita240796, could we split this out into a separate list of jobs, and a separate section in the Admin jobs page? I think these will be long-lasting jobs and it makes sense conceptually to separate them so that they are not all mixed up with the rest.

OK to do this in a separate PR if you prefer, just thought I'd mention it. (If you decide not to do it in this PR, maybe file an issue or do something else to keep track of the action item.)

Thanks!

Reply:
I will be doing this in a separate PR. Filed #6887

Reply:
Also, filed a separate issue for running backend tests split on basis of test classes #6888

Reply:
Thanks!

Just a note, I'll need this to be done before I run the tests in prod. This helps minimize serious errors since currently it's quite easy to click the button for the wrong job.

Reply:
Sure, I will make this change before asking you to run the jobs. Thanks!